robottelo/host_helpers/satellite_mixins.py

```diff
@@ -168,6 +191,23 @@ def publish_content_view(self, org, repo_list):
         content_view = content_view.read()
         return content_view
 
+    def move_pulp_archive(self, org, export_message):
+        """
+        Moves exported archive(s) and its metadata into import directory,
+        sets ownership, returns import path
+        """
+        self.execute(f'mv {PULP_EXPORT_DIR}/{org.name} {PULP_IMPORT_DIR}')
+        self.execute(f'chown -R pulp:pulp {PULP_IMPORT_DIR}')
```

Comment:
if you're running these two back to back without checking the status, it's likely better to just run them both in a single execute.